core/jobs_registry.py

```diff
@@ -88,7 +88,8 @@
     user_jobs_one_off.UserProfilePictureOneOffJob,
     user_jobs_one_off.UsernameLengthDistributionOneOffJob,
     exp_jobs_one_off.MathExpressionInputInteractionOneOffJob,
-    exp_jobs_one_off.MultipleChoiceInteractionOneOffJob
+    exp_jobs_one_off.MultipleChoiceInteractionOneOffJob,
+    exp_jobs_one_off.VoiceoverDurationSecondsOneOffJob
```

Comment:
Add the last 3 items in the list in sorted order.

Reply:
ok

Reply:
So re-sort this entire list?

Reply:
Yep, you can do that too.

Reply:
ok

Reply:
done